### PR TITLE
feat(mgmt): add pseudo restart http listener

### PR DESCRIPTION
**Pull request by wwhai**
Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

File: lib-ce/emqx_management/src/emqx_mgmt_cli.erl [hunk 1]
```diff
@@ -744,3 +756,11 @@ listener_identifier(Protocol, ListenOn) ->
         ID ->
             ID
     end.
+
+%% TODO Maybe redesign this api.
+restart_emqx_management(Identifier) ->
+    emqx_ctl:print("Listener ~p not support restart!~n", [Identifier]).
```
Comment (zmstone) [inline lib-ce/emqx_management/src/emqx_mgmt_cli.erl:762]:
```suggestion
    emqx_ctl:print("Management listener restart is not supported~n", []).
```

File: lib-ce/emqx_management/src/emqx_mgmt_cli.erl [hunk 2]
```diff
@@ -744,3 +756,11 @@ listener_identifier(Protocol, ListenOn) ->
...
+
+restart_emqx_dashboard(Identifier) ->
+    emqx_ctl:print("Listener ~p not support restart!~n", [Identifier]).
```
Comment (zmstone) [inline lib-ce/emqx_management/src/emqx_mgmt_cli.erl:765]:
```suggestion
    emqx_ctl:print("Dashboard listener restart is not supported~n", []).
```

**Comment by zmstone** [comment]
we also need to stub the rest API to return HTTP code `403`

**Comment by zmstone** [comment]
Please rebase one the latest master.

File: lib-ce/emqx_management/src/emqx_mgmt_cli.erl (hunk 1 above)
Comment (HJianBo) [reply lib-ce/emqx_management/src/emqx_mgmt_cli.erl:762]: Is this planned for the next release?
I think it is possible to implement a real command to restart the HTTP listener. Why are we including it in this PR?

**Pull request closed without merging**